pkg/reconciler/route/resources/certificate_test.go

```diff
@@ -84,3 +85,62 @@ func TestMakeCertificates(t *testing.T) {
 		t.Errorf("MakeCertificate (-want, +got) = %v", diff)
 	}
 }
+
+func TestMakeCertificates_CorrectMetadata(t *testing.T) {
```

Comment:
A very minor nit: we can name this test `TestMakeCertificates_FilterLastAppliedAnno`, or we can combine this test with `TestMakeCertificates` with 
```
var testcases = []struct {
		name       string
		route       *v1alpha1.Route
    want        []*netv1alpha1.Certificate
	}{
......
}
```

Reply:
Sure, thanks.